src/tests/multihost/alltests/test_krb_access_provider.py

```diff
@@ -78,20 +79,16 @@ def test_0001_k5login_empty(multihost):
         multihost.client[0].run_command("truncate -s 0 /var/log/secure", raiseonerr=False)
         client = sssdTools(multihost.client[0])
         client.clear_sssd_cache()
-
-        users = ['foo3', 'foo4']
-        ssh = client.auth_from_client(users[0], 'Secret123')
-        ssh2 = client.auth_from_client(users[1], 'Secret123')
+        with pytest.raises(Exception):
+            check_login_client(multihost, "foo3", "Secret123")
+        check_login_client(multihost, "foo4", "Secret123")
+        time.sleep(3)
         secure_log_str = multihost.client[0].get_file_contents("/var/log/secure").decode('utf-8')
-
         file = f'/var/log/sssd/sssd_{ds_instance_name}.log'
         sssd_log_str = multihost.client[0].get_file_contents(file).decode('utf-8')
 
         multihost.client[0].run_command("truncate -s 0 /home/foo3/.k5login", raiseonerr=False)
```

Comment:
This cleanup will not happen if the check_login_client fails on exception.
The file should be probably deleted instead of just shortening it.